examples/zookeeper.py

```diff
@@ -29,9 +29,10 @@ def __init__(self):
 
 
 	async def _on_zk_ready(self, event_name, zkcontainer):
-		path = self.ZkContainer.ZooKeeperPath + "/hello"
-		await self.ZkContainer.ZooKeeper.ensure_path(path)
-		print("The path in Zookeeper has been created.")
+		if zkcontainer == self.ZkContainer:
```

Comment:
This deserves a comment.

Reply:
@ateska @eliska-n 

This condition should be more robust if you are trying to subscribe to "ZooKeeperContainer.started!" outside of the module, ex:  situations where multiple containers exists and you are interested in a specific container only.